src/controllers/healthcheck-controller.ts

```diff
@@ -26,7 +26,7 @@ export default class HealthcheckController {
 
       return res.status(OK).send()
     } catch (err) {
-      logger.err(err);
+      logger.err(`Performing healtcheck failed: ${err.message()}`);
```

Comment:
`Healtcheck failed: ${err.message()}`

the "performing" is a bit confusing/misleading

Reply:
Oh sorry @stbrody didn't see full context here. But your latest change makes sense.